package.json

```diff
@@ -10,20 +10,20 @@
     "test": "npm run test -ws --if-present",
     "start:library": "npm start -w library",
     "start:example": "./start-example.sh",
-    "start:map-example": "EXAMPLE=map run-p start:library start:example",
-    "start:map-with-markers-example": "EXAMPLE=map-with-markers run-p start:library start:example",
-    "start:multiple-maps-example": "EXAMPLE=multiple-maps run-p start:library start:example",
-    "start:geocoding-service-example": "EXAMPLE=geocoding-service run-p start:library start:example",
-    "start:places-service-example": "EXAMPLE=places-service run-p start:library start:example",
-    "start:places-service-with-element-example": "EXAMPLE=places-service-with-element run-p start:library start:example",
-    "start:places-autocomplete-widget-example": "EXAMPLE=places-autocomplete-widget run-p start:library start:example",
-    "start:directions-service-example": "EXAMPLE=directions-service run-p start:library start:example",
-    "start:distance-matrix-service-example": "EXAMPLE=distance-matrix-service run-p start:library start:example",
-    "start:elevation-service-example": "EXAMPLE=elevation-service run-p start:library start:example",
-    "start:max-zoom-service-example": "EXAMPLE=max-zoom-service run-p start:library start:example",
-    "start:places-autocomplete-service-example": "EXAMPLE=places-autocomplete-service run-p start:library start:example",
-    "start:street-view-panorama-map-example": "EXAMPLE=street-view-panorama-map run-p start:library start:example",
-    "start:street-view-panorama-element-example": "EXAMPLE=street-view-panorama-element run-p start:library start:example",
+    "start:map-example": "cross-env EXAMPLE=map run-p start:library start:example",
```

Comment:
Please add `cross-env` to the `devDependencies` in this package.json.

Reply:
☝️

Reply:
Oopsi